### DISCUSSION TITLE
pagelle

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#544] [Idee e proposte]
Perchè i genitori non possono stampare le pagelle dell'anno precedente?E soprattutto perché insieme alla pagella finale non è visibile anche quella del primo quadrimestre/trimestre

**Comment by giadaschiavone** [comment]
Ciao Luigi, potrebbe dipendere anche dal fatto che negli anni precedenti il registro è stato cambiato e/o il sito. Spero di esserti stata d'aiuto!

**Comment by trinko** [comment]
Per quanto riguarda lo storico degli esiti ho già risposto [qui](https://github.com/iisgiua/giuaschool/discussions/543).

Per la pagella del primo periodo (trimestre/quadrimestre): queste sono VISIBILI ai genitori, basta cliccare sull'apposito menu di scelta periodo (in alto).
